pricing-page-exercise/salih/index.html

```diff
@@ -1,13 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./styles.css">
-  <title>Pricing page Exercise</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>getbootstrap Pricing Page</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-  
+    <nav>
+        <div class="nav-container">
+            <div class="company-name">
+                <span>Company Name</span>
```

Comment:
Usually, a link element (<a></a>) is used for a logo or company name in the navigation bar. This link should always take the user to the home page. I recommend to change <span></span> here to <a></a>.

Reply:
Okay, i made it `a `element with a link.